### PR TITLE
Package updates to master - to use node 16.4.0

### PR DESCRIPTION
**Pull request by gregmercer**

### CODE REVIEW

**Comment by sherakama** [comment]
@gregmercer, a couple of quick bits. We are using `npm` to manage this repo vs `yarn` and you'll need to update the `.nvmrc` file to ensure the Netlify environment gets the correct version of node.

**Comment by gregmercer** [comment]
@sherakama Sorry I forgot to add the .nvmrc. I've updated now and removed the yarn.lock file.

**Comment by sherakama** [comment]
Thanks @gregmercer  but it looks like the build is failing because package-lock.json needs to be updated as well.

**Comment by gregmercer** [comment]
Oh... yes sure. Added lock, updated packages.json and nvmrc files. Used yarn just to get the dependencies sorted. :)

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-11.14.0
+14.15.4
```
Comment (sherakama) [inline .nvmrc:1]: @gregmercer didn't you want this to be Node version 16?

You don't have to specify the minor or patch number either if you just want to require `16` that would also be ok.

Comment (gregmercer) [reply .nvmrc]: @sherakama 
I've been able to update this to 15.14.0. Using npm I haven't been able to go up to 16 without getting this error:
error: no template named 'remove_cv_t' in namespace 'std'; did you mean 'remove_cv'?
Using yarn was the only way I could seem to be on node 16 and get things to work.
I've run across a few folks saying basically this:
It seems to be caused by an incompatibility [between node-sass and Node 16] https://stackoverflow.com/questions/67241196/error-no-template-named-remove-cv-t-in-namespace-std-did-you-mean-remove

Comment (sherakama) [reply .nvmrc]: Hmm. Yeah, It looks like we will have to go to Next 12 to get node-sass support at ^6.0.0 

That sounds like a much bigger refactor/upgrade than we need. Node 14 would probably make more sense as the odd valued releases get removed from support after 6 months and the even valued releases get put into LTS

https://nodejs.org/en/about/releases/

**Comment by sherakama** [comment]
Hey @gregmercer 

I've finally gotten around to the big upgrade lift. You can find it in https://github.com/SU-SWS/decanter-web/pull/142

This is also going to see a big lift in the core decanter project: https://github.com/SU-SWS/decanter/pull/922

Thanks for this PR but I am going to close it in favor of #142

**Pull request closed without merging**